docs/docs/faq.md

```diff
@@ -76,6 +76,49 @@ The `conda` config is located in the `/home/{user}/.condarc` file. You can chang
 
 You may find that the pods hosting your environment get full over time, prompting you to clear them out. To delete old builds of your environment on conda-store, click the "delete" button in the conda-store UI.
 
+## What are Preemptible and Spot instances on Nebari?
```

Comment:
```suggestion
## What are preemptible and spot instances on Nebari?
```

We prefer sentence case for headings for the docs :) Ref:https://www.nebari.dev/docs/community/style-guide#capitalization

Reply:
Actually, I think "How to use preemptible and spot instances on Nebari?" might be a better fit here, what do you think?